src/drawer.multicanvas.js

```diff
@@ -210,11 +221,11 @@ export default class MultiCanvas extends Drawer {
         let lastEntry = this.canvases[this.canvases.length - 1];
 
         // wave
-        lastEntry.wave.parentElement.removeChild(lastEntry.wave);
+        lastEntry.wave.parentElement.removeChild(lastEntry.wave.proxiedElement);
 
         // progress
         if (this.hasProgressCanvas) {
-            lastEntry.progress.parentElement.removeChild(lastEntry.progress);
+            lastEntry.progress.parentElement.removeChild(lastEntry.progress.proxiedElement);
```

Comment:
I'm still a little confused why you use `proxiedElement` property only here, and not in other places in multicanvas. Ideally we don't deal with this attribute at all..

Reply:
I explain in this comment: https://github.com/katspaugh/wavesurfer.js/pull/2195/files#diff-c016f970dc3c8172ada63105abcd2b2f0e837a3c24c7592df6046dacd1bd4e0dR53

To expand a bit: Unfortunately, it doesn't seem possible to add/remove proxied elements to the DOM directly. [This blog post](https://hacks.mozilla.org/2015/07/es6-in-depth-proxies-and-reflect/) explains it briefly:

> There are limits to the fidelity of the illusion. You’ll find that `proxy !== target`. And a proxy will sometimes flunk type checks that the target would pass. Even if a proxy’s target is a DOM Element, for example, the proxy isn’t _really_ an Element; so something like `document.body.appendChild(proxy)` will fail with a `TypeError`.

I encountered this myself when testing this. So I have to use `proxiedElement` where a proxied element would touch the DOM directly.

Reply:
I see. Perhaps `domElement` or `proxiedDOMElement` would better reflect the nature of the attribute. `proxiedElement` is too vague for me to be honest. I would also prefer to split the regions plugins changes into a separate PR that gets merged once this PR is merged. It would make it easier to read the changes made to core. thoughts?

Reply:
Renamed to `domElement` per your request. The code you asked about should work, as the proxy passes through `set` to the underlying DOM element: https://github.com/katspaugh/wavesurfer.js/pull/2195/files#diff-c016f970dc3c8172ada63105abcd2b2f0e837a3c24c7592df6046dacd1bd4e0dR91